test/dependencies.js

```diff
@@ -22,6 +22,26 @@ describe('dependencies', () => {
     })
   })
 
+  describe('rpmSupportsBooleanDependencies', () => {
+    afterEach(() => {
+      sinon.restore()
+    })
+
+    const testParts = [2, 3, 4]
+    testParts.forEach(parts => {
+      const extraParts = Array(parts - 1).join('extra-part ')
+      const versionString = `RPM-Version ${extraParts}4.15.1`
+
+      it(`parses RPM version output "${versionString}" (with ${parts} parts) correctly`, async () => {
+        sinon.stub(dependencies, 'spawnRpmBuild').resolves(versionString)
+        const stub = sinon.stub(dependencies, 'rpmVersionSupportsBooleanDependencies')
+
+        await dependencies.rpmSupportsBooleanDependencies(null)
+        sinon.assert.calledWith(stub, '4.15.1')
+      })
+    })
+  })
```

Comment:
I don't think the tests need to be this meta. It's better to test against real-world data anyway.
```suggestion
  describe('getRpmVersion', () => {
    it('parses version output from RPM < 4.15', async () => {
      const versionOutput = 'RPM version 4.14.2.1\n'
      // stub spawn
      expect(dependencies.getRpmVersion()).to.equal('4.14.2.1')
    })
    
    it('parses version output from RPM >= 4.15', async () => {
      const versionOutput = 'RPM-Version 4.15.1\n'
      // stub spawn
      expect(dependencies.getRpmVersion()).to.equal('4.15.1')
    })
  })
```